Cubical/Foundations/Prelude.agda

```diff
@@ -65,6 +66,10 @@ cong : (f : (a : A) → B a) (p : x ≡ y) →
 cong f p i = f (p i)
 {-# INLINE cong #-}
 
+congS : ∀ {B : Type ℓ} → (f : A → B) (p : x ≡ y) → f x ≡ f y
```

Comment:
What does the `S` stand for? Also what is the point of having this simpler form?

Reply:
`S` stands for simple (simply typed), just like `P` seems to stand for dependent? Using `congS` instead of `cong` can simplify some unification problems leading to the resolution of some metavariables that would otherwise remain unsolved.

Reply:
Ah ok, I see. I think someone explained this to me before, but I forgot about it. Can you please add a comment about it?